src/js/summernote.js

```diff
@@ -164,54 +169,81 @@ define([
      * ```
      *   
      * @member $.fn
-     * @param {Object} options reference to $.summernote.options
-     * @returns {this}
+     * @param {Object|String} options reference to $.summernote.options
+     * @return {this}
      */
-    summernote: function (options) {
-      // extend default options
-      options = $.extend({}, $.summernote.options, options);
+    summernote: function () {
+      var type = $.type(list.head(arguments));
+
+      // if first argument's type is string, first arguments is module and method
+      var moduleAndMethod, args;
+      if (type === 'string') {
+        moduleAndMethod = list.head(list.from(arguments));
+        args = list.tail(list.from(arguments));
+      }
+
+      // extend default options with custom user options
+      var options = type === 'object' ? list.head(arguments) : {};
+      options = $.extend(true, $.summernote.options, options);
 
       // Include langInfo in options for later use, e.g. for image drag-n-drop
       // Setup language info with en-US as default
-      options.langInfo = $.extend(true, {}, $.summernote.lang['en-US'], $.summernote.lang[options.lang]);
+      options.langInfo = $.extend(true, $.summernote.lang['en-US'], $.summernote.lang[options.lang]);
```

Comment:
(Same as above)
Won't this override the default `$.summernote.lang['en-US']`?
See [jQuery extend doc ](http://api.jquery.com/jquery.extend/) and #708